package.json

```diff
@@ -83,8 +83,7 @@
     "requirejs": "gulp requirejs",
     "generateDocumentation": "gulp generateDocumentation",
     "instrumentForCoverage": "gulp instrumentForCoverage",
-    "eslint": "gulp eslint",
-    "eslint-watch": "gulp eslint-watch",
+    "eslint": "eslint ./**/*.js ./**/*.html --cache",
```

Comment:
Maybe add `--quiet` to suppress warnings? I think that was the `gulp-eslint` default